config/main.py

```diff
@@ -1179,7 +1169,17 @@ def reload(db, filename, yes, load_sysinfo, no_service_restart, disable_arp_cach
         config_db.connect()
         client = config_db.get_redis_client(config_db.CONFIG_DB)
         client.flushdb()
+
         if load_sysinfo:
+            command = "{} -j {} -v DEVICE_METADATA.localhost.hwsku".format(SONIC_CFGGEN_PATH, file)
+            proc = subprocess.Popen(command, shell=True, text=True, stdout=subprocess.PIPE)
+            cfg_hwsku, err = proc.communicate()
+            if err:
+                click.echo("Could not get the HWSKU from config file, exiting")
+                sys.exit(1)
+            else:
+                cfg_hwsku = cfg_hwsku.decode('UTF-8').strip()
```

Comment:
>'UTF-8' [](http://example.com/codeflow?start=45&length=7)

Not a blocking issue, but you can remove it and use the default value.

Reply:
Modified as requested.